mantidproject/mantid
# Fix threshold filter in splatterplot view

Description of work.

This adds cell data to the vtkUnstructuredGrid generated by the SplatterPlot filter, which is required by the vtkThreshold filter. I also updated the icon to be consistent with the VSI standard view and ParaView. Lastly I removed an incorrect and unnecessary tooltip in the VSI standard view. 

I created a issue #18339 to find a better solution that doesn't require adding cell data to the vtkUnstructuredGrid.

**To test:**



1. Load a MDEventWorkspace and open in in the VSI.
2. (if necessary) Switch to the SplatterPlot view
3. Click the threshold filter button, change the min and max values and click apply.
4. Check if the output changes.

Fixes #18330.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
